pyscf/pbc/dft/numint.py

```diff
@@ -75,10 +75,12 @@ def eval_ao_kpts(cell, coords, kpts=None, deriv=0, relativity=0,
         ao_kpts: (nkpts, [comp], ngrids, nao) ndarray
             AO values at each k-point
     '''
+    from types import SimpleNamespace
     if kpts is None:
         if 'kpt' in kwargs:
-            sys.stderr.write('WARN: KNumInt.eval_ao function finds keyword '
-                             'argument "kpt" and converts it to "kpts"\n')
+            logger.warn(SimpleNamespace(verbose=verbose),
+                        'KNumInt.eval_ao function finds keyword '
```

Comment:
`SimpleNamespace(verbose=verbose)` fails with `NameError: name 'verbose' is not defined` . Change this to `cell`.